### PR TITLE
Jetpack Licenses: Add links to license activation screen

### PR DESCRIPTION
**Pull request by robertf4**
Fixes 1201509629272450-as-1201532056430595

#### Changes proposed in this Pull Request:

This adds some helpful product-specific links to the screen shown after a license is successfully activated.

image

#### Jetpack product discussion

p1HpG7-e3y-p2

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

For each of the Jetpack products and plans do the following:

1. On a new site, visit `/wp-admin/admin.php?page=jetpack#/license/activation` and activate a license.
2. Verify that the success page shows with an appropriate external link, and a button that goes to the `my-plan` page.

### CODE REVIEW

File: projects/js-packages/licensing/components/activation-screen-success-info/product-details/index.js [hunk 1]
```diff
@@ -100,9 +105,9 @@ const JetpackProductDetails = props => {
 	};
 
 	return (
-		<div>
+		<div className="jp-license-activation-screen-success-info--product-details">
 			<h1>
-				{ productInfoMap[ productGroup ].title }{ ' ' }
+				{ productInfoMap[ productGroup ].title }&nbsp;
```
Comment (rcanepa) [inline projects/js-packages/licensing/components/activation-screen-success-info/product-details/index.js:110]: `&nbsp;` prevents the icon from wrapping to the next line alone.

**Review by rcanepa — APPROVED** [review]
These changes look good to me.

Jetpack products other than Jetpack VideoPress and Jetpack Anti-spam have an external link to an appropriate section on the cloud, and there is a fallback button that leads users to the My Plan section (keeping them in WP Admin).

The changes also look good on mobile.

**Review by samiff — DISMISSED** [review]
Tested well for me 👍

**Review by samiff — APPROVED** [review]

**Comment by anomiex** [comment]
Hi! You seem to have ignored the bot that told you to commit D72192-code. Forgetting to do that means that future PRs might fail Fusion because they expect the changes made by this PR to be present. If you need help, feel free to ask.

I went ahead and did it for you now, because exactly that has happened. r237747-wpcom

**Pull request merged**